calcom/cal.com
# feat(lexicalEditor): custom enter-key plugin

## What does this PR do?



- Fixes https://github.com/calcom/cal.com/issues/19318
- [CAL-3713] Line breaks not reflecting in the event type description




https://github.com/user-attachments/assets/bdd5d75e-27a6-4283-9033-12d49ff6c920



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

Can be tested on the latest production code. 

Note: when typing continusuoly the text will go to next line. However that possible doesn't makes it a line break in the description ui. It should be only visible when we manually enter while typing.
